packages/manager/modules/mfa-enrollment/src/mfa-enrollment.html

```diff
@@ -0,0 +1,36 @@
+<div class="mfa-enrollment-page container-fluid h-100 position-absolute overflow-auto">
+    <div class="row minh-100 justify-content-md-center">
+        <div class="col-md-7 p-4 p-md-5 bg-white minh-100">
+            <div class="p-0 p-md-5">
+                <div class="text-center mb-3">
+                    <img
+                        data-ng-src="{{$ctrl.DEFAULT_ASSET}}"
+                        srcset="{{$ctrl.DEFAULT_ASSET2X}} x2, {{$ctrl.DEFAULT_ASSET3X}} x3"
+                        class="img-fluid"
+                        alt="" />
+                </div>
+
+                <h1 class="oui-heading_4" data-translate="mfa_enrollment_title"></h1>
+                <p data-translate="mfa_enrollment_info1"></p>
+                <p data-translate="mfa_enrollment_info2"></p>
+
+                <div class="mx-auto my-5 float-right">
+                    <oui-button class="m-2"
+                        data-ng-if="!$ctrl.forced"
+                        data-on-click="$ctrl.goBack()">
+                        <span data-translate="mfa_enrollment_cancel"></span>
+                    </oui-button>
+
+                    <a class="oui-button oui-button_primary oui-button_icon-right m-2"
+                       data-ng-href="{{$ctrl.mfaUrl}}"
+                       data-track-on="click"
```

Comment:
Then you have a peer dependency with [ng-at-internet](https://www.npmjs.com/package/ng-at-internet) package I guess.